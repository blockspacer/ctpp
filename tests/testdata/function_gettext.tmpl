// Start

_: <TMPL_var _("Unknown message")>
GETTEXT: <TMPL_var GETTEXT("Unknown message")>
_: <TMPL_var _("Test message")>
GETTEXT: <TMPL_var GETTEXT("Test message")>
_ n1: <TMPL_var _("%d file", "%d files", n1)>
_ n2: <TMPL_var _("%d file", "%d files", n2)>
_ n5: <TMPL_var _("%d file", "%d files", n5)>
GETTEXT n1: <TMPL_var GETTEXT("%d file", "%d files", n1)>
GETTEXT n2: <TMPL_var GETTEXT("%d file", "%d files", n2)>
GETTEXT n5: <TMPL_var GETTEXT("%d file", "%d files", n5)>
_ n1: <TMPL_var _("unknown singular message", "unknown plural message", n1)>
_ n2: <TMPL_var _("unknown singular message", "unknown plural message", n2)>
GETTEXT n1: <TMPL_var GETTEXT("unknown singular message", "unknown plural message", n1)>
GETTEXT n2: <TMPL_var GETTEXT("unknown singular message", "unknown plural message", n2)>

Plural: 1 <TMPL_var russian_plural(1,'рубль', 'рубля', 'рублей')>
Plural: 2 <TMPL_var russian_plural(2,'рубль', 'рубля', 'рублей')>
Plural: 3 <TMPL_var russian_plural(3,'рубль', 'рубля', 'рублей')>
Plural: 4 <TMPL_var russian_plural(4,'рубль', 'рубля', 'рублей')>
Plural: 5 <TMPL_var russian_plural(5,'рубль', 'рубля', 'рублей')>
Plural: 6 <TMPL_var russian_plural(6,'рубль', 'рубля', 'рублей')>
Plural: 7 <TMPL_var russian_plural(7,'рубль', 'рубля', 'рублей')>
Plural: 8 <TMPL_var russian_plural(8,'рубль', 'рубля', 'рублей')>
Plural: 9 <TMPL_var russian_plural(9,'рубль', 'рубля', 'рублей')>
Plural: 10 <TMPL_var russian_plural(10,'рубль', 'рубля', 'рублей')>
Plural: 11 <TMPL_var russian_plural(11,'рубль', 'рубля', 'рублей')>
Plural: 12 <TMPL_var russian_plural(12,'рубль', 'рубля', 'рублей')>
// End.
