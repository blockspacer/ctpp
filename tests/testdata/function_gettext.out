// Start

_: Unknown message
GETTEXT: Unknown message
_: Тестовое сообщение
GETTEXT: Тестовое сообщение
_ n1: %d файл
_ n2: %d файла
_ n5: %d файлов
GETTEXT n1: %d файл
GETTEXT n2: %d файла
GETTEXT n5: %d файлов
_ n1: unknown singular message
_ n2: unknown plural message
GETTEXT n1: unknown singular message
GETTEXT n2: unknown plural message

Plural: 1 рубль
Plural: 2 рубля
Plural: 3 рубля
Plural: 4 рубля
Plural: 5 рублей
Plural: 6 рублей
Plural: 7 рублей
Plural: 8 рублей
Plural: 9 рублей
Plural: 10 рублей
Plural: 11 рублей
Plural: 12 рублей
// End.
